projects/ui-particles-angular/src/lib/gio-form-json-schema/wrappers/gio-currency-prefix-wrapper.component.scss

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2023 The Gravitee team (http://gravitee.io)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.currency-prefix {
+  margin-left: 8px;
+  color: rgb(0 0 0 / 60%);
```

Comment:
We do not use direct colors, but always colors from the color palettes. I think we can just delete it and keep the default color. no ?